homeassistant/components/darksky/sensor.py

```diff
@@ -574,6 +576,15 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the Dark Sky sensor."""
+    async_create_issue(
```

Comment:
This integration already creates an issue:

<https://alerts.home-assistant.io/alerts/dark_sky/>

Reply:
So we should just remove it?

Reply:
I think that would be OK; it will probably break before shipping the next release (if they actually take it down... They have extended it many times before).

Reply:
There was no note about prolonging it somewhere and also users on Reddit noticing that the api got unavailable some time so probably they might have already started early to shut things down timely. 

I will change it to remove the integration instead.